substra/sdk/client.py

```diff
@@ -110,6 +110,20 @@ def set_user(self):
                 })
                 self.client.set_config(self._current_profile, self._profile_name)
 
+    def update_permissions_field(self, data, permissions_field='permissions'):
```

Comment:
This function has no real reason to be part of the Client class: it does not interact with the server, it modifies a dict object. So it could be moved outside of the client class.

This function must be private: API users should not be aware of this function.